frontend/rest-frontend/src/main/java/com/alexkoveckiy/frontend/rest/security/Filter.java

```diff
@@ -24,20 +28,23 @@
     @Autowired
     private TokenHandler tokenHandler;
 
-    private TokenAuthentication tokenAuthentication;
+    private Authentication authentication;
 
     public Filter() {
-        this.tokenAuthentication = new TokenAuthentication();
+        this.authentication = new UsernamePasswordAuthenticationToken(null, null);
```

Comment:
@Alexey007gold Do you think Authentication object will be same for all requests from all users ?